actions/sha256.go

```diff
@@ -24,8 +24,19 @@ import (
 	"net/http"
 )
 
-func SHA256FromURI(uri string) string {
-	resp, err := http.Get(uri)
+type RequestModifierFunc func(request *http.Request) *http.Request
+
+func SHA256FromURI(uri string, f RequestModifierFunc) string {
+	req, err := http.NewRequest("GET", uri, nil)
+	if err != nil {
+		panic(fmt.Errorf("unable to create GET %s request\n%w", uri, err))
```

Comment:
I too have used the panic on each error check "pattern" for simple "scripts" in go. But once it is complex enough that we are extracting a shared package I think it is time to introduce actual error handling.